apis/r/R/SOMACollection.R

```diff
@@ -1,9 +1,11 @@
 #' SOMA Collection
 #'
+#' [lifecycle: experimental]
+#'
 #' @description Contains a key-value mapping where the keys are string names
 #' and the values are any SOMA-defined foundational or composed type, including
 #' `SOMACollection`, `SOMADataFrame`, `SOMADenseNdArray`, `SOMASparseNdArray`
-#' or `SOMAExperiment`.
+#' or `SOMAExperiment`.  [lifecycle: experimental]
```

Comment:
Redundant lifecycles added or this intentional? (I'm not sure R docs are ultimately rendered...)

Reply:
Looking at other places, we're only adding in `@description`, so the first one can be removed